### ISSUE TITLE
--reckless mode -- store in superdataset's .git/config?

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#4752]
If you install/clone/whatever a subdataset as reckless and then `uninstall` it, and then install it -- datalad (0.13.1.dev55) would do a full install.  
ATM we store information that is a reckless clone only within that subdataset's `.git/config` as
```
[datalad "clone"]
	reckless = ephemeral
```
(see #4749 for a gotcha with plain git) so super-dataset has no record on how that dataset is/was installed.
I guess we could just consider that a feature, but I wonder if we should store also in the superdataset's .git/config information about subdataset "previous" (and thus preferable) reckless installation .  

If we do, we would need to fix handling of `--reckless` to be able to explicitly disable it (#4751).

**Comment by mih** [comment]
I cannot really follow the "i did it before, hence I want it again"-logic. If everything was splendid, why did you uninstall the subdataset? Is this based on a real use case?

**Comment by yarikoptic** [comment]
Yes. For Yoda stole local processing I might install a number of subdatasets, some of which from local ones in reckless mode. I do processing, save the results and uninstall all subdatasets since no longer needed. In case of recomputation or additional computation, I need to install this subdatasets again, eg via datalad rerun. Without additional annotation it wouldn't know which ones to install in reckless mode
